src/main/java/com/ably/kafka/connect/mapping/DefaultMessageSinkMapping.java

```diff
@@ -31,7 +31,7 @@ public DefaultMessageSinkMapping(@Nonnull ChannelSinkConnectorConfig config, @No
     @Override
     public Message getMessage(SinkRecord record) {
         final Message message = messageFromRecord(record);
-        message.id = String.format("%d:%d:%d", record.topic().hashCode(), record.kafkaPartition(), record.kafkaOffset());
+        //message.id = String.format("%d:%d:%d", record.topic().hashCode(), record.kafkaPartition(), record.kafkaOffset());
```

Comment:
Commented out since it was causing errors with REST BATCH API.

Reply:
Can we add a `// TODO` so that we know that this needs to be resolved so that we can support idempotency again?

Reply:
Added ToDO.